neck_yaw_to_neck_pitch: [-0.0105, 0, 0.0362]
neck_pitch_to_camera_center: [-0.005, 0, 0.0315]
